src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1427,7 +1441,7 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
             & ,fv3filenamegin(it)%dynvars,fv3filenamegin(it))
             call gsi_fv3ncdf_read(grd_fv3lam_tracer_ionouv,gsibundle_fv3lam_tracer_nouv &
             & ,fv3filenamegin(it)%tracers,fv3filenamegin(it))
-            if( if_model_dbz )then
+            if( if_model_dbz .or. if_model_fed )then
```

Comment:
better to test nphyvario  or even test if grd_fv3lam_phyvar_ionovu is allocated.

Reply:
@TingLei-NOAA 
Use 'if( nphyvario3d > 0 )then' here. 
There is an allocated test in line about #1231 before the bundle was created.